ee/agent/storage/sqlite/keyvalue_store_sqlite.go

```diff
@@ -19,20 +19,23 @@ import (
 	_ "modernc.org/sqlite"
 )
 
-type storeName int
+type StoreName int
```

Comment:
This needs to be exported?

Reply:
Exported for use in the `pkg/log/sqlitelogger` package, looks like